.ci/publish-aws.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+#
+# Publishes the created artifacts from ./dev-utils/make-distribution.sh to AWS as AWS Lambda Layers in every region.
+# Finalized by generating an ARN table which will be used in the release notes.
+#
+# AWS_FOLDER is used for temporary output of publishing layers used to create the arn table. (Optional)
+# ELASTIC_LAYER_NAME is the name of the lambda layer e.g. elastic-apm-python-ver-3-44-1 for the git tag v3.44.1 (Required)
+
+export AWS_FOLDER=${AWS_FOLDER:-.aws}
+FULL_LAYER_NAME=${ELASTIC_LAYER_NAME:?layer name not provided}
+ALL_AWS_REGIONS=$(aws ec2 describe-regions --output json --no-cli-pager | jq -r '.Regions[].RegionName')
+
+rm -rf "${AWS_FOLDER}"
+mkdir -p "${AWS_FOLDER}"
+
+# Delete previous layers
+for region in $ALL_AWS_REGIONS; do
+  layer_versions=$(aws lambda list-layer-versions --region="${region}" --layer-name="${ELASTIC_LAYER_NAME}" | jq '.LayerVersions[].Version')
+  echo "Found layer versions for ${FULL_LAYER_NAME} in ${region}: ${layer_versions:-none}"
+  for version_number in $layer_versions; do
+    echo "- Deleting ${FULL_LAYER_NAME}:${version_number} in ${region}"
+    aws lambda delete-layer-version \
+        --region="${region}" \
+        --layer-name="${FULL_LAYER_NAME}" \
+        --version-number="${version_number}"
+  done
+done
+
+
+zip_file="./build/dist/elastic-apm-python-lambda-layer.zip"
+
+for region in $ALL_AWS_REGIONS; do
+  echo "Publish ${FULL_LAYER_NAME} in ${region}"
+  publish_output=$(aws lambda \
+    --output json \
+    publish-layer-version \
+    --region="${region}" \
+    --layer-name="${FULL_LAYER_NAME}" \
+    --description="AWS Lambda Extension Layer for the Elastic APM Python Agent" \
+    --license="Apache-2.0" \
```

Comment:
As far as I see the `apm-agent-python` uses a different [license](https://github.com/elastic/apm-agent-python/blob/main/LICENSE). But I don't know what's the one for the AWS lambda, can we clarify this?

Reply:
Good point. It's copied from apm-aws-lambda. I mistakenly assumed it will be the same.

Reply:
Any suggestions @basepi @beniwohli ?

Reply:
https://github.com/elastic/apm-agent-python/commit/30729a05b33e39568457de56bcccc12a29fe681c